src/SelfService/Infrastructure/Api/Capabilities/CapabilityController.cs

```diff
@@ -135,8 +140,17 @@ public async Task<IActionResult> CreateNewCapability([FromBody] NewCapabilityReq
             );
             return ValidationProblem(statusCode: StatusCodes.Status409Conflict);
         }
-
+        
         var capability = await _capabilityRepository.Get(capabilityId);
+
+        if (request.Invitees != null)
+        {
+            await _invitationApplicationService.CreateCapabilityInvitations(
+                invitees: request.Invitees,
+                inviter: userId,
+                capability: capability
+            );
+        }
```

Comment:
We probably need to explicitly handle the situation where creating the capability went well but inviting went wrong. Should we roll back the capability creation, should we send back a partial success code (does that even exist?)

Reply:
I am open to suggestions here.
There is no such thing as a partial success response code.

We can create a new response object and have the frontend verify that we invited the people it expected us to invite.
We could also choose to fail.
This will always be the issue with joined operation, and I do not have a good or preferred solution for it.